packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -2096,7 +2176,56 @@ curl -v -X POST \
 }
 ```
 
-#### Assign a Custom Role to a User or Group
+#### Assign a Role to a Client Application
+
+<ApiOperation method="post" url="/oauth2/v1/clients/${clientId}/roles" />
+
+Assigns a Role to a Group
```

Comment:
Is this correct? Where did "group" come from?
```suggestion
Assigns a Role to a Client
```

Reply:
Copy paste error :)